packages/office-ui-fabric-react/src/components/Fabric/__snapshots__/Fabric.test.tsx.snap

```diff
@@ -1,6 +1,58 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Fabric renders correctly 1`] = `
+exports[`Fabric renders a Fabric component correctly 1`] = `
+<div
+  className=
+      ms-Fabric
+      {
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+        color: #323130;
+        font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
+        font-size: 14px;
+        font-weight: 400;
+      }
+      & button {
+        font-family: inherit;
+      }
+      & input {
+        font-family: inherit;
+      }
+      & textarea {
+        font-family: inherit;
+      }
+>
+  test
+</div>
+`;
+
+exports[`Fabric renders a Fabric component with applyTheme correctly 1`] = `
+<div
+  className=
+      ms-Fabric
+      {
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+        color: #323130;
+        font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
+        font-size: 14px;
+        font-weight: 400;
```

Comment:
I would expect `background-color` to appear here. Is that an incorrect assumption?

Reply:
Hmm yes now that I look at these again, they kind of all look identical even though the DOM I can see, when I test locally by doing `applyTheme` and `applyThemeBody` to exising examples that each is changing the DOM differently.

Reply:
I believe this is because the applyTheme conditional in Fabric.styles.ts isn't working properly

Reply:
Fixed this! It was because the props `applyTheme` and `applyThemeToBody` weren't properly being pulled from getClassNames in the base.